cert-manager/src/components/certificateRequests/List.tsx

```diff
@@ -0,0 +1,56 @@
+import { ResourceListView } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+import { useEffect, useState } from 'react';
+import { CertificateRequest } from '../../types/certificateRequest';
+import { isCertManagerInstalled } from '../../utils';
+import { NotInstalledBanner } from '../common/NotInstalledBanner';
+
+export function CertificateRequestsList() {
+  const [isManagerInstalled, setIsManagerInstalled] = useState(false);
```

Comment:
this part (useState + useEffect) is repeated in each `List` component. please extract it into a it's own hook.

this starts with value false which will render NotInstalledBanner by default, even if it is installed. I'd recommend adding additional field `isManagerCheckLoading` that will keep track whether the check is still loading. this will avoid flashing "NotInstalledBanner" 

also this check is loaded every time you enter the page, it should be fetched once and remember the value

Reply:
Created a separate hook for this.